asmcli/asmcli.sh

```diff
@@ -611,6 +617,8 @@ OPTIONS:
   -l|--cluster_location  <LOCATION>
   -n|--cluster_name      <NAME>
   -p|--project_id        <ID>
+  --kubeconfig           <KUBECONFIG>
+  --context              <CONTEXT>
```

Comment:
I imagine these two are going to be used fairly often, what do you think about also accepting `--kc` and `--ctx`?

Reply:
i kept these to match **kubectl** and similar cli flags for the two values.

Reply:
Right, I meant

```
--kubeconfig|--kc \t\t <KUBECONFIG>
--context|--ctx \t\t <CONTEXT>
```

Reply:
done!